### ISSUE TITLE
Instance: The fact that new messages (non-reply messages) have appeared is not announced to users | WCAG 2.2 Violation - There are changes in status in the interface that are not announced to users.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#47]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

### Screenshot

![Screenshot showing the addition of a new messages to the list of messages within the interface. The fact that new messages (non-reply messages) like this have appeared is not announced to assistive technology users.](https://github.com/user-attachments/assets/9a06c6ac-9908-44f2-bf14-47e2ebce6d4a)

### Code Reference

```
<!-- The `.chat-messages` element (where new messages are added) -->
<div id="chat-comments-app" class="chat-comments-wrapper" data-initialized="true">
	<div class="chat-comments-container">
		[…]
		<div class="chat-messages">[…]</div>
		[…]
	</div>
</div>
```

#### Further information:

1. 

### Instances

1.
